sympy/matrices/determinant.py

```diff
@@ -344,7 +344,7 @@ def _adjugate(M, method="berkowitz", dotprodsimp=None):
     sympy.matrices.common.MatrixCommon.transpose
     """
 
-    return M.cofactor_matrix(method=method, dotprodsimp=dotprodsimp).transpose()
+    return _cofactor_matrix(M, method=method, dotprodsimp=dotprodsimp).transpose()

```

Comment:
I know this is not you, but `sympy` code should avoid capitalized variables. `mat` would be preferable over `M`.

Reply:
I am generally variable-name agnostic and try to follow established coding convention as you can see with other variables but in this case I did specifically select `M` to stand out like a `self` which it essentially is and also to differentiate it from other uses of `mat` like in `_row_reduce` where it is not actually a matrix but a list, to avoid confusion. If this bothers you too much though I can change it, just say so.